### DISCUSSION TITLE
Oven features?

### DISCUSSION THREAD

**Discussion by jaybe-dk** [astrandb/miele#397] [General]
Hi,

I have installed the integration and it seems to work.

I have a Miele oven and it seems the only sensor I have available in HA is whether the door is open or not; no sensors for temperature, on/off status, program etc.

Am I missing something or is it as designed?

/Jaybe

**Comment by astrandb** [comment]
Please turn on the oven and make a diagnostic download from the device page. Then drag the file to this issue and I will take a look at the possibilities.

**Reply by PianSom** [reply]
Hi - is this still on your to-do list? I have had a new Miele oven for a few months, and have no useful sensors (eg current oven temperature). Happy to post any info you need to see.

**Reply by astrandb** [reply]
It should show something... Please turn on the oven and make a screen-shot and a diagnostic download from the device overview page and drag them into this discussion thread and I will have a look.

**Reply by PianSom** [reply]
Many apologies - user error. I actually have a Neff oven, not a Miele. :)

**Comment by jaybe-dk** [comment]
Thanks for the very quick response @astrandb ! I turned on the oven and restarted HA - and now I have many more sensors and switches. No need to investigate :)

**Comment by ElectricZebraSpeaker** [comment]
Hey jaybe-dk. Are you able to provide a screenshot or something of the sensors you have available now?

I will be getting a Miele oven soon but I am curious what the capabilities and sensors are through HA. Thanks!

**Reply by jaybe-dk** [reply]
Sure, happy to help - see attached. Not all sensors are available in all programs. The oven is great BTW 😊
![IMG_0521](https://github.com/astrandb/miele/assets/80651051/0d40d9b9-12e0-4f43-be2a-c6d87c0d253c)
![IMG_0522](https://github.com/astrandb/miele/assets/80651051/563b4950-a19a-4c7e-8c7e-607d92f4cac7)

**Reply by jaybe-dk** [reply]
…and here’s an initial dashboard while a chicken is roasting.
![IMG_0523](https://github.com/astrandb/miele/assets/80651051/1c89d38f-9456-4fd1-92fb-352010e591d0)
